config/cashier.php

```diff
@@ -0,0 +1,18 @@
+<?php
```

Comment:
Not sure if a config file will be needed. I added one early when I thought it'd be necessary.